flutter/android.mk

```diff
@@ -0,0 +1,94 @@
+# Copyright 2020-2021 The MLPerf Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+##########################################################################
+
+.PHONY: flutter/android
+flutter/android: flutter/android/libs flutter/prepare-flutter
```

Comment:
I got an error running `make flutter/android`
```
➜  mobile_app_open git:(164/flutter-backends-integration) make flutter/android
...
mkdir -p flutter/android/app/src/main/jniLibs/arm64-v8a
cp -f --target-directory flutter/android/app/src/main/jniLibs/arm64-v8a \
		bazel-bin/mobile_back_tflite/cpp/backend_tflite/libtflitebackend.so \
		 \
		 \
		 \
		 \
		bazel-bin/flutter/cpp/flutter/libbackendbridge.so
cp: illegal option -- -
usage: cp [-R [-H | -L | -P]] [-fi | -n] [-apvXc] source_file target_file
       cp [-R [-H | -L | -P]] [-fi | -n] [-apvXc] source_file ... target_directory
make: *** [flutter/android/libs] Error 64
```

Command `make flutter/ios` can be run successful, though.

Reply:
Should be fixed now